### ISSUE TITLE
No accounts found  on Unicove login

### ISSUE THREAD

**Issue by cuvialabs** [greymass/unicove#325]
Testing the Metamask Snap.

1.   Installed the SNAP
![image](https://github.com/user-attachments/assets/c01c4da0-7dbd-4a72-aba1-5dd5341760b4)

2. Tried to login into Unicove and got this
![image](https://github.com/user-attachments/assets/4293aa39-8305-49fb-a675-6c1b9bca1f4c)

I dont see the EOS network in the available list of networks.
![image](https://github.com/user-attachments/assets/0676082c-7096-4569-8bbe-e453ed4ab867)

This is done using Metamask  12.9.2 using Brave Browser  1.7.3.104 and Chrome  131.0.6778.205 on Windows 10

**Comment by cuvialabs** [comment]
Solved,    When I go back to the orginial install page,  I can now see these options.   
![image](https://github.com/user-attachments/assets/c0c027b5-dc69-4396-aa8e-84237c77601c)

**Comment by cuvialabs** [comment]
How do I add an existing EOS account?

**Comment by aaroncox** [comment]
Yeah - using the "Login" will prompt the install, but won't go through the creation process. Maybe we need to add a button or something there to loop you back to account creation if you head down that path.

As for existing accounts, we decided not to allow arbitrarily importing keys - so the only way to move an existing account to use MetaMask is to add/change the keys on the account to match MetaMask keys. We're looking at ways so a tool can be used to make this happen but they aren't built yet. 

You can do it manually if you'd like. If you enable Advanced Mode on Unicove, and then visit the MetaMask page, it'll show you both the owner and active public keys derived from MetaMask. Then you'd take the active public key and add it as the active key on the existing account (either adding it as a 2nd active key, or replacing your existing active key). Then MetaMask would be in control of the account.
